docs/computing/lumi-vs-mahti.md

```diff
@@ -0,0 +1,55 @@
+# How does LUMI-C differ from Mahti?
+
+[LUMI-C](https://docs.lumi-supercomputer.eu/computing/systems/lumic/) consists of 1536 AMD CPU nodes (2x64 cores each) with a performance of 5.6 petaflops. The node count of [Mahti](systems-mahti.md) is similar, 1404 AMD CPU nodes (2x64 cores each) corresponding to a performance of 7.5 Petaflops. Although the systems are very alike based on hardware, core count and performance, there are important differences related to access and use which are highlighted on this page.
+
+## Accessing and SSH keys
+
+To access LUMI, you need to first [create a LUMI-specific project](../accounts/how-to-create-new-project.md#how-to-create-finnish-lumi-projects). Furthermore, accessing LUMI is only possible using SSH keys, meaning that you cannot use passwords to connect through SSH like on Mahti. For instructions on how to generate an SSH key pair and uploading the public key to [MyCSC](https://my.csc.fi/), see [Setting up SSH keys](connecting.md#setting-up-ssh-keys) and the [Get started with LUMI](https://docs.lumi-supercomputer.eu/firststeps/getstarted/) pages.
+
+## Finite time projects
```

Comment:
Not only that they have finite time, but in general that they are a different type of projects, and cannot use same projects for lumi and national stuff. Both available from my.csc.fi though